chapter_introduction/index_vn.md

```diff
@@ -1270,15 +1271,15 @@ with the benefit (or harm) associated with it:
 -->
 
 Bây giờ giả sử ta đã xây dựng một bộ phân loại và huấn luyện nó để dự đoán liệu một cây nấm có độc hay không dựa trên ảnh chụp.
-Giả sử bộ phân loại phát hiện chất độc đưa ra $P(y=\mathrm{nấm độc}|\mathrm{bức ảnh}) = 0.2$.
+Giả sử bộ phân loại phát hiện chất độc đưa ra $P(y=\mathrm{\text{nấm độc}}|\mathrm{\text{bức ảnh}}) = 0.2$.
 Nói cách khác, bộ phân loại này chắc chắn rằng $80\%$ cây này *không phải* nấm độc.
 Dù vậy, đừng dại mà ăn nhé.
 Vì việc có bữa tối ngon lành không đáng gì so với rủi ro $20\%$ sẽ chết vì nấm độc.
 Nói cách khác, hậu quả của *rủi ro không chắc chắn* nghiêm trọng hơn nhiều so với lợi ích thu được.
-Ta có thể nhìn việc này một cách hợp thức hơn.
+Ta có thể nhìn việc này theo khía cạnh lý thuyết.
 Về cơ bản, ta cần tính toán rủi ro kỳ vọng mà mình sẽ gánh chịu, ví dụ, ta nhân xác suất xảy ra kết quả đó với lợi ích (hoặc hậu quả) đi liền tương ứng:
 
-$$L(\mathrm{hành động}| x) = E_{y \sim p(y| x)}[\mathrm{mất mát}(\mathrm{hành động},y)].$$
+$$L(\mathrm{\text{hành động}}| x) = E_{y \sim p(y| x)}[\mathrm{\text{MấtMát}}(\mathrm{\text{hành động}},y)].$$
```

Comment:
Tại sao "mất mát" lại viết hoa và viết liền nhỉ?

Reply:
Ở đây do nó là hàm toán học `loss()` nên nếu viết cách ra thì nhìn hơi kỳ `mất mát()` nên có hai cách là sử dụng gạch dưới `mât_mát()` hoặc viết hoa mỗi chữ cái đầu. Hiện tại em đang dùng cách viết hoa

Reply:
Ý của @rootonchair mình thấy hay đấy. Mình sửa thế này mn thấy sao?
```suggestion
$$L(\mathrm{\text{hành động}}| x) = E_{y \sim p(y| x)}[\mathrm{\text{mất_mát}}(\mathrm{\text{hành động}},y)].$$
```

Reply:
![image](https://user-images.githubusercontent.com/23548268/72205129-1c30ba80-34b2-11ea-8de1-4d90090e7b8b.png)
![image](https://user-images.githubusercontent.com/23548268/72205138-3c607980-34b2-11ea-8e5a-b3151277a53c.png)

hmm, có vẻ viết thường dễ nhìn hơn.

Reply:
Ủa, cái hình này em lấy ở đâu ra? Trên web của mình à?

Reply:
Cái này hơi tiểu tiết em ạ, anh thấy trên pdf đọc vẫn ổn:
![](https://i.ibb.co/58R7KDC/Screen-Shot-2020-01-11-at-10-10-36-AM.png)